lib/oli_web/templates/activity_bank/index.html.eex

```diff
@@ -4,10 +4,6 @@
   <script type="text/javascript" src="<%= Routes.static_path(@conn, "/js/" <> script) %>"></script>
 <% end %>
 
-<div id="editor" class="container"/>
-
-<script>
-
-  const params = <%= raw(Jason.encode!(@context)) %>;
-  window.oliMountApplication(document.getElementById('editor'), params);
-</script>
+<div id="editor" class="container">
+  <%= ReactPhoenix.ClientSide.react_component("Components.ActivityBank", @context) %>
```

Comment:
You can embed any valid react component like this, as long as that component has been registered to some place we can access it, such as window.Components

Reply:
NIT: I like this optional step mentioned [here](https://github.com/geolessel/react-phoenix#5-optional-import-the-module-into-your-views-for-less-typing) and just do `react_component` in the templates